qforce/qm/orca.py

```diff
@@ -0,0 +1,624 @@
+import os.path
+
+from colt import Colt
+import numpy as np
+from ase.units import Hartree, mol, kJ, Bohr
+#
+from .qm_base import WriteABC, ReadABC
+from ..elements import ATOM_SYM
+
+
+class Orca(Colt):
+    _user_input = """
+
+    charge_method = esp :: str :: [cm5, esp]
+    
+    # QM method to be used for geometry optimisation
+    opt_method = XTB2 :: str
+    # For high-throughput use XTB2
+    # For accuracy use r2SCAN-3c
+    
+    # QM method to be used for hessian calculation
+    # Note: The accuracy of this method determines the accuracy of bond, 
+    # angle and improper dihedral.
+    hess_method = B3LYP D4 def2-TZVP def2/J RIJCOSX
+    
+    # QM method to be used for charge derivation
+    # Note: Method chosen according to the standard RESP procedure.
+    c_method = HF 6-31G* :: str
```

Comment:
charge_method?

Reply:
@selimsami I think we cannot use charge_method as it is used for `charge_method = esp :: str :: [cm5, esp]`

Reply:
Hmm having c_method and charge_method is a bit ambiguous.
I'd leave charge_method as it is, since it corresponds to other QM methods. (missing description btw)
Maybe an alternate name for c_method? Can't think of something nice right now. A bit long but maybe all methods can change to: 
qm_method_hessian
qm_method_charge
qm_method_...

Any other ideas?

Reply:
I think a bit long is fine. 
I will change it to 
qm_method_opt
qm_method_hessian
qm_method_charge
qm_method_sp